exercises/practice/hello-world/hello-world.el

```diff
@@ -1,4 +1,9 @@
-(defun hello ()  -*- lexical-binding: t; -*-
+;;; hello-world.el --- Hello World exercise (exercism)  -*- lexical-binding: t; -*-
```

Comment:
What is the minimal amount of code you can use here?

Reply:
I'm not sure I understand the question

Reply:
The function declaration was there for a reason, in order to make the amount of steps the user needs to take as minimal as possible.

We recommend even prefilling the return value with "Goodbye, Mars!" And have the user change that to Hello, World! as this exercise is meant to get your computer in shape, not to be an actual challenge.

Reply:
@SleeplessByte I think the function declaration is there, but just in another part of the file.

Reply:
Ah. Doesn't change my other comment then 😁😁

Reply:
You're referring to addings more comments to explain what's happening?

Reply:
Maybe something like this? (mostly quoting from the Emacs Manual)

```elisp
;;; hello-world.el --- Hello World exercise (exercism)  -*- lexical-binding: t; -*-

;;; Commentary:

;;; Emacs has conventions for using special comments in Lisp libraries that give structured information about the library.
;;; Using a standard format for these items makes it easier for tools (and people) to extract the relevant information.
;;; For more information, see
;;; https://www.gnu.org/software/emacs/manual/html_node/elisp/Library-Headers.html

;;; Code:
```